src/main/java/co/org/osso/api/CalculatorService.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CalculatorService {
+
+
+    public List getFibonacci(Integer number) {
```

Comment:
Let's be explicit of what kind of List we're returning: `List<Integer>`. I can't think of a use case when I'd use just `List` on its own.